Boulder-Cryogenic-Quantum-Testbed/scresonators
# (mostly) Untested fitting code from OO refactor (#206)

* Update README.md

* Update tests.yml

For some reason the other implementation was failing to find python files

* Update tests.yml

* Update tests.yml

* Start of refactor only focusing on fitting for now

* Untested addition of fit

* Updating setup

* Add version file

* Add version file

* Update inits

* Update inits

* Check for colinearity

* Consistency with fit_method and correct parameters for fit

* Consistency with fit_method and correct parameters for fit

* Actually fitting git status!

* MC fit working with short chain

* Bootstrapping confidence intervals for robustness

---------
